acos_client/v30/slb/hm.py

```diff
@@ -82,8 +82,12 @@ def _set(self, action, name, mon_method, interval, timeout, max_retries,
                 "override-ipv4": ipv4
             }
         }
-        if method:
-            params['monitor']['method'][mon_method]['url-type'] = method
+        if mon_method in ['http', 'https']:
+            if method:
+                params['monitor']['method'][mon_method]['url-type'] = method
+        if mon_method in ['https']:
```

Comment:
We can collapse this to be:

```python
if mon_method in ['http', 'https']:
    if method:
        params['monitor']['method'][mon_method]['url-type'] = method
    if mon_method == 'https' and disable_sslv2:
        params['monitor']['method'][mon_method]['disable-sslv2hello'] = disable_sslv2
```

Reply:
I thought about this but http hm does not have a disable-sslv2hello parameter.

If we try to create a http hm using this parameter, it will fail. It will also cause all subsequent http health monitor to fail as well. 
This is the same when trying to create a TCP hm and using the method parameter.

The only way to be able to create http hm again is by opening a new python session.

Attached is a document of the issue that I saw.
[Wrong_attribute_issue.txt](https://github.com/a10networks/acos-client/files/5278098/Wrong_attribute_issue.txt)